variables.tf

```diff
@@ -200,3 +200,9 @@ variable "enable_adminer" {
   default     = false
   type        = bool
 }
+
+variable "enable_ec2_detailed_monitoring" {
+  description = "Enables/disables detailed monitoring for EC2 instances"
+  type        = bool
+  default     = false
```

Comment:
This is a reusable module (i.e. it is a "public" repo and uses semantic versioning) so this setting should remain `true`.

However, academically speaking, since it is pre-1.0 it actually could be changed to false and break backward compatibility with appropriate warnings listed in the release notes. But for consistency and simplicity, let's keep it at `true`.

Reply:
Okey @briskt  will update it to true. :+1: